components/Modal/index.d.ts

```diff
@@ -1,55 +1,61 @@
-import { ReactNode, TouchEvent, MouseEventHandler } from 'react';
+import { ReactNode, TouchEvent, MouseEventHandler, FC, HTMLAttributes } from 'react';
 
 export interface ModalProps {
   children?: ReactNode[] | ReactNode;
-  onClick?: (e?: MouseEvent | TouchEvent) => void;
+  onClick?: (e?: MouseEvent | TouchEvent) => void; // braking change
```

Comment:
why the comment?

Reply:
The default type of prop "e" (event) is MouseEvent only, TouchEvent is unnecessary. Thinking about removing this unnecessary type, it is possible that this generates breakchange as it may be that some application wrongly defines the type of the callback for the onClick prop